installing/installing_aws/installing-aws-default.adoc

```diff
@@ -29,7 +29,10 @@ program.
 ====
 * If you use a firewall, you must
 xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
-
+* If you do not allow the system to manage identity and access management (IAM),
+then you can xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam[manually
+create IAM credentials]. In environments where the cloud IAM APIs are not
+reachable, the Cloud Credential Operator can be disabled prior to installation.
```

Comment:
For the last sentence, suggest: "Manual mode can also be used in environments where the cloud IAM APIs are not reachable."

Lets avoid any mention of calling the operator "disabled", that was a bad call on my part, we expect in manual mode it will do increasingly more and I don't want people thinking it's actually disabled and then it starts reporting problems they don't want to deal with.